src/winzone/WindowsTimeZoneIDs.jl

```diff
@@ -34,30 +35,67 @@ function compile(xml_file::AbstractString, translation_file::AbstractString)
         end
     end
 
+    return translation
+end
+
+function compile(xml_file::AbstractString, translation_file::AbstractString)
     open(translation_file, "w") do fp
-        serialize(fp, translation)
+        serialize(fp, compile(xml_file))
```

Comment:
It shouldn't change between patch releases, but I'm not sure we have good tests ensuring that. It definitely changes between minor releases, and during the course of nightlies.